jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/EppsteinKShortestPath.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * (C) Copyright 2019-2019, by Semen Chudakov and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.alg.shortestpath;
+
+import org.jgrapht.Graph;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.interfaces.KShortestPathAlgorithm;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Implementation of the Eppstein`s algorithm for finding
+ * $k$ shortest path between two vertices in a graph.
+ *
+ * <p>
+ * This implementation can only be used for directed simple graphs.
+ *
+ * <p>
+ * The algorithm is originally described in
+ * <a href="https://www.ics.uci.edu/~eppstein/pubs/Epp-SJC-98.pdf">this paper</a>.
```

Comment:
Please add a proper reference. Here's where the article is published: https://epubs.siam.org/doi/10.1137/S0097539795290477

Reply:
Seems like neither of the citation formats on the web page is suitable to be used in the documentation. I have taken the citation from [ACM site](https://dl.acm.org/citation.cfm?id=299886).